tests/integration/GitHubSnapshot.test.ts

```diff
@@ -0,0 +1,62 @@
+import {context, getMocks} from "../mocks";
+const { setData, restoreInitialData, mocks } = getMocks();
+let requestArgs: any;
+delete mocks["@actions/tool-cache"]
+delete mocks["@actions/exec"]
+const mockCreator = mocks["@actions/github"]
```

Comment:
This would probably be easier to read if we had empty lines between sections (for example, after imports)